browser-test/src/applicant/questions/northstar_id.test.ts

```diff
@@ -0,0 +1,262 @@
+      await applicantQuestions.answerIdQuestion('abcde')
+      await applicantQuestions.clickContinue()
+
+      await expect(
+        page.getByText('Error: Must contain only numbers.'),
+      ).toBeVisible()
+    })
+  })
+
+  test.describe('multiple id questions', () => {
+    const programName = 'Test program for multiple ids'
+
+    test.beforeEach(async ({page, adminQuestions, adminPrograms}) => {
+      await loginAsAdmin(page)
+
+      await adminQuestions.addIdQuestion({
+        questionName: 'my-id-q',
+      })
+      await adminQuestions.addIdQuestion({
+        questionName: 'your-id-q',
+      })
+
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.editProgramBlockWithOptional(
+        programName,
+        'Optional question block',
+        ['my-id-q'],
+        'your-id-q', // optional
+      )
+      await adminPrograms.publishAllDrafts()
+
+      await logout(page)
+    })
+
+    test('with both id inputs submits successfully', async ({
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.applyProgram(
+        programName,
+        /* northStarEnabled= */ true,
+      )
+      await applicantQuestions.answerIdQuestion('12345', 0)
+      await applicantQuestions.answerIdQuestion('67890', 1)
+      await applicantQuestions.clickContinue()
+
+      await applicantQuestions.expectReviewPage(/* northStarEnabled= */ true)
+    })
+
+    test('with unanswered optional question submits successfully', async ({
+      applicantQuestions,
+    }) => {
+      // Only answer second question. First is optional.
+      await applicantQuestions.applyProgram(
+        programName,
+        /* northStarEnabled= */ true,
+      )
+      await applicantQuestions.answerIdQuestion('67890', 1)
+      await applicantQuestions.clickContinue()
+
+      await applicantQuestions.expectReviewPage(/* northStarEnabled= */ true)
+    })
+
+    test('with first invalid does not submit', async ({
+      page,
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.applyProgram(
+        programName,
+        /* northStarEnabled= */ true,
+      )
+      await applicantQuestions.answerIdQuestion('abcde', 0)
+      await applicantQuestions.answerIdQuestion('67890', 1)
+      await applicantQuestions.clickContinue()
+
+      await expect(
+        page.getByText('Error: Must contain only numbers.'),
+      ).toBeVisible()
+    })
+
+    test('with second invalid does not submit', async ({
+      page,
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.applyProgram(
+        programName,
+        /* northStarEnabled= */ true,
+      )
+      await applicantQuestions.answerIdQuestion('67890', 0)
+      await applicantQuestions.answerIdQuestion('abcde', 1)
+      await applicantQuestions.clickContinue()
+
+      await expect(
+        page.getByText('Error: Must contain only numbers.'),
+      ).toBeVisible()
+    })
+
+    test('has no accessiblity violations', async ({
```

Comment:
**question (non-blocking):** I wonder if it's worth it to test accessibility both here and in the single id question test above.  I'm having trouble imagining how the number of id questions would make a difference to accessibility.  What do you think?

Reply:
The legacy tests only validate accessibility in the 2 question case, so I deleted the other one above.